AlgoPhantomBackend/AlgoPhantomBackend/settings.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-from .local_settings import *
+# from .local_settings import *
```

Comment:
I was getting some error while running the server. After commenting it worked. Sorry for that.

Reply:
> You can't comment on this as it imports the settings config.

Hey brother, can i use "try except" for this?

Reply:
No need to be sorry :) 
You should check the setup.md first, it will help you to set up and after that, you will get the idea of how to remove these errors.
Still, you get any error or trouble feel free to ask :)

Reply:
Sure, I'll do it by tomorrow 😊